maintainers/maintainer-list.nix

```diff
@@ -7522,6 +7522,11 @@
     githubId = 5121426;
     name = "Albert Safin";
   };
+  xfoxawy = {
+    email = "xfoxawy@gmail.com";
+    github = "xfoxawy";
+    name = "Mohammed Kamal";
+  }
```

Comment:
please add githubId, forgot where the directions are to find this value, but should be easy to find.